### PR TITLE
edited script to retry till the DNS is configured and added docker-compose example

### PR DESCRIPTION
**Pull request by chicco785**
few improvements:
* edited script to retry till the DNS is configured (sleep time of 5 sec in the loop)
* added docker-compose example (including healthchecks, tested with docker stack command)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+version: '3.2'
+
+services:
+  # recall to sudo sysctl -w vm.max_map_count=262144 on swarm nodes
+  elasticsearch:
+    image: martel/docker-swarm-elasticsearch:5.4.0
```
Comment (a-goryachev) [inline docker-compose.yml]: Looks like here should be `agoryachev/docker-swarm-elasticsearch:5.4.0`

Comment (chicco785) [reply docker-compose.yml]: hi, you are totally right :) of course having to test it, I replicated the image somewhere else, i will update right away.

**Comment by chicco785** [comment]
it should be ok now.
actually, I was thinking also to an alternative that could be adopted to inject the script without having to rebuild the image.
in other swarm projects I do leverage on SECRETS to pass configuration files. it may be an option worth exploring.

**Comment by a-goryachev** [comment]
@chicco785 Thanks for contribution!

**Pull request merged**